javascript/javascript2/week3/The fastest presser in this realm.js

```diff
@@ -0,0 +1,51 @@
+"use strict";
+
+// counter for L
+let totalL = 0;
+const elL = document.querySelector("#countL");
+const pressedL = (e) => {
```

Comment:
```suggestion
const pressedL = (event) => {
```

But here super good with the function that just takes care of this!